src/docs-developer-guide.rst

```diff
@@ -100,13 +100,13 @@ It is helpful to keep the CSS file organized.
 There are several sections in the :file:`custom.css` file:
 
 - Styling for rst roles and directives
-- Responsive css
+- Responsive CSS
```

Comment:
You can apply the fix here as well: 
- https://github.com/opendatakit/docs/blob/master/src/_static/css/custom.css#L244
- https://github.com/opendatakit/docs/blob/master/src/_static/css/custom.css#L255